src/brasil/gov/portal/upgrades/v10900/handler.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from brasil.gov.agenda.config import PROJECTNAME as AGENDAPROJECTNAME
+from brasil.gov.portal.config import PROJECTNAME
+from collective.cover.controlpanel import ICoverSettings
+from plone import api
+from plone.registry.interfaces import IRegistry
+from Products.GenericSetup.tool import UNKNOWN
+from zope.component import getUtility
+
+import logging
+
+
+logger = logging.getLogger(PROJECTNAME)
+
+
+def atualiza_estilos_cover(context):
+    """Atualiza estilos da configuração do collective.cover caso os mesmos
+    tenham sido perdidos. Ver:
+    https://github.com/collective/collective.cover/issues/465
+    """
+    registry = getUtility(IRegistry)
+    settings = registry.forInterface(ICoverSettings)
+    old_styles = settings.styles
+    new_styles = set([
+        'Amarelo|amarelo',
+        'Azul Claro - borda|azul-claro-borda',
+        'Azul Claro Saude|azul-claro',
+        'Azul Escuro Turismo|azul-escuro',
+        'Azul Governo|azul',
+        'Azul Petroleo Defesa Seguranca|azul-petroleo',
+        'Azul Piscina|azul-piscina',
+        'Azul Turquesa - borda|azul-turquesa-borda',
+        'Azul Turquesa|azul-turquesa',
+        'Bege - borda|bege-borda',
+        'Bege|bege',
+        'Dourado Cultura|dourado',
+        'Fio separador|fio-separador',
+        'Laranja - borda|laranja-borda',
+        'Laranja Cidadania Justica|laranja',
+        'Link Externo|link-externo',
+        'Lista Horizontal|lista-horizontal',
+        'Lista Vertical|lista-vertical',
+        'Marrom Claro Economia Emprego|marrom-claro',
+        'Marrom Infraestrutura|marrom',
+        'Padrao|padrao',
+        'Roxo - borda|roxo-borda',
+        'Roxo Ciencia Tecnologia|roxo',
+        'Verde Claro Meio Ambiente|verde-claro',
+        'Verde Escuro Educacao|verde-escuro',
+        'Verde Esporte|verde',
+    ])
+    union_styles = sorted(old_styles.union(new_styles))
+    settings.styles = set(union_styles)
+    logger.info('Estilos do cover atualizados')
+
+
+def instala_profile_agenda(context):
+    """Instala o profile do brasil.gov.agenda se ele não estiver instalado.
+    Em versões antigas do brasil.gov.portal o profile do brasil.gov.agenda não
+    era instalado. Ver:
+    https://github.com/plonegovbr/brasil.gov.portal/issues/154#issuecomment-78988918
+    """
+    ps = api.portal.get_tool('portal_setup')
```

Comment:
the context is the `portal_setup` tool; there's no need to get it, just change the parameter name to increase readability.